modules/python/init.zsh

```diff
@@ -34,6 +34,51 @@ if (( ! $+commands[python] && ! $+commands[pyenv] )); then
   return 1
 fi
 
+function _python-workon-cwd {
+    if [[ -z "$WORKON_CWD" ]]; then
```

Comment:
I believe it was originally there to prevent it running twice before finishing. I don't see how it is useful as a cwd hook call. Going to remove it.